backend/src/game/game.ts

```diff
@@ -0,0 +1,49 @@
+import { Server, Socket } from 'socket.io';
+import { CreateGameDto } from '@dtos/game/create-game.dto';
```

Comment:
```suggestion
import { CreateGameDto, GameStateDto } from '@dtos/game';
```
To apply this patch, one has to have an `index.ts` in th `@dtos/game` directory.